src/wallet/wallet2.cpp

```diff
@@ -8521,42 +8522,42 @@ static lns_prepared_args prepare_tx_extra_loki_name_system_values(wallet2 const
     }
 
     boost::optional<std::string> failed;
-    std::vector<cryptonote::COMMAND_RPC_LNS_NAMES_TO_OWNERS::response_entry> response = wallet.lns_names_to_owners(request, failed);
+    *response = wallet.lns_names_to_owners(request, failed);
```

Comment:
There needs to be a null check on this when passing it as a pointer. If given you fill out their struct, otherwise fill a local out a local copy.